src/ResourceManager/ResourceManager/Commands.ResourceManager/Cmdlets/Implementation/TestAzureResourceCmdlet.cs

```diff
@@ -131,6 +131,7 @@ protected override void OnProcessRecord()
         /// </summary>
         private void RunCmdlet()
         {
+            this.WriteWarning("This cmdlet is being deprecated. ");
```

Comment:
IMO we should change this to follow the standard PS cmdlets deprecation message as follows - "The Test-AzureResource cmdlet is deprecated and will be removed in a future release.”